c_src/ffmpeg_video_compositor/video_compositor.c

```diff
@@ -0,0 +1,124 @@
+#include "video_compositor.h"
```

Comment:
Please provide documentation for functions, especially for `create` and `apply_filter`.

Reply:
Done